From 286b259401115a0b8494bc0d528ef01d18f63ce8 Mon Sep 17 00:00:00 2001
From: Shaveta Leekha <shaveta@freescale.com>
Date: Tue, 18 Oct 2011 17:51:33 +0000
Subject: [PATCH 47/92] rf: Add gpios to be used by AD PHY device in adphy node

Extracted from 913x_WUSDK_REL_0.9.tar.gz vendor drop.

Signed-off-by: Shaveta Leekha <shaveta@freescale.com>

GPIO2 pin of gpio1 controller is used to reset the AD9361 device.
GPIO17 and 18 pins of gpio controller are used for band switching.
Integrated-by: Jiang Bin <bin.jiang@windriver.com>
---
 arch/powerpc/boot/dts/psc9131rdb.dts |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/psc9131rdb.dts b/arch/powerpc/boot/dts/psc9131rdb.dts
index 21b2953..f833f9d 100644
--- a/arch/powerpc/boot/dts/psc9131rdb.dts
+++ b/arch/powerpc/boot/dts/psc9131rdb.dts
@@ -275,6 +275,9 @@
 				rfphy0: ad9361_phy@0{
 					compatible = "fsl,espi-ad_phy","ad9361";
 					reg = <0>;
+					gpios = <&gpio1 2 1   /* RESET */
+						 &gpio 17 1
+						 &gpio 18 1>;
 					linux,modalias = "ad9361_phy";
 					spi-max-frequency = <20000000>;
 					spi-cpha;
-- 
1.7.0

